### PR TITLE
feat(ELY-412): Create custom external roles directly from main module

### PR DESCRIPTION
**Pull request by askoriy**

### CODE REVIEW

File: main.tf [hunk 1]
```diff
@@ -131,6 +131,14 @@ module "gcr_project_roles" {
   sa_depends_on = module.services_sa.email
 }
 
+module "custom_external_roles" {
+  source = "./modules/external-roles"
+
+  roles_map     = var.custom_external_roles
+  project_id    = module.project_factory.project_id
+  sa_depends_on = module.services_sa.email
```
Comment (anakovt) [inline main.tf]: why does it depend on services_sa.email? can't you add these custom roles to any specified SA?

File: modules/external-roles/README.md [hunk 2]
```diff
@@ -18,6 +18,20 @@ service-account2@project_id.iam.gserviceaccount.com:
 
 See [example](../../examples/external-roles) of using the module
 
+If `project_id` variable provided then only service name may be passed to map instead of whole service account:
+
+```
+service1:
```
Comment (anakovt) [inline modules/external-roles/README.md]: is this related to our services specified in the project.yaml?

Comment (askoriy) [reply modules/external-roles/README.md]: It's not related to our services, it means only left part of service account can be speficied

Comment (anakovt) [reply modules/external-roles/README.md]: okay, but shouldn't we add like this?
(in the README service is a bit confusing)
```
roles_map = {
  service_account_1 = {
    project_id_1 = [
        role1,
        role2
],
     project_id_2 = [
        role1,
        role2
],
},
  service_account_2 ....
}
```

**Review by anakovt — APPROVED** [review]

**Pull request merged**